.github/actions/update-releases-md/index.js

```diff
@@ -0,0 +1,111 @@
+const actionUtils = require('../action-utils.js');
+
+async function run() {
+    const [core, github] = await actionUtils.installAndRequirePackages("@actions/core", "@actions/github");
+
+    const releasesDataFile = core.getInput("releases_json_file", { required: true });
+    const outputFile = core.getInput("releases_md_file", { required: true });
+
+    const repoOwner = github.context.payload.repository.owner.login;
+    const repoName = github.context.payload.repository.name;
+
+    try {
+        const releasesData = JSON.parse(await actionUtils.readFile(releasesDataFile));
+
+        const releasesMdContent = generateReleasesMdContent(releasesData, repoOwner, repoName);
+
+        await actionUtils.writeFile(outputFile, releasesMdContent);
+    } catch (error) {
+        core.setFailed(error);
+    }
+}
+
+function generateReleasesMdContent(releasesData, repoOwner, repoName) {
+    let supportedReleasesTable = '';
+    let previewReleasesTable = '';
+    let outOfSupportReleasesTable = '';
+
+    for (const releaseKey of releasesData.supported) {
+        supportedReleasesTable += `${generateTableRow(releasesData.releases[releaseKey], repoOwner, repoName, true)}\n`;
+    }
+
+    for (const releaseKey of releasesData.preview) {
+        previewReleasesTable += `${generateTableRow(releasesData.releases[releaseKey], repoOwner, repoName, false)}\n`;
+    }
+
+    for (const releaseKey of releasesData.unsupported) {
+        outOfSupportReleasesTable += `${generateTableRow(releasesData.releases[releaseKey], repoOwner, repoName, true)}\n`;
+    }
+
+    let content =`
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Freleases)
+
+# Releases
+
+`;
+
+    if (supportedReleasesTable.length > 0) {
+        content += `## Supported versions\n\n${generateTableHeader(true)}\n${supportedReleasesTable}\n\n`;
+    }
+
+    if (outOfSupportReleasesTable.length > 0) {
+        content += `## Out of support versions\n\n${generateTableHeader(true)}\n${outOfSupportReleasesTable}\n\n`;
+    }
+
+    if (previewReleasesTable.length > 0) {
+        content += `## Preview versions\n\n${generateTableHeader(false)}\n${previewReleasesTable}\n\n`;
+    }
+
+
+    return content;
+}
+
+function generateTableHeader(rtmVersions) {
+    let headers = ['Version'];
+    if (rtmVersions) {
+        headers.push('Original Release Date', 'Latest Patch Version', 'Patch Release Date', 'End of Support');
+    } else {
+        headers.push('Release Date');
```

Comment:
nit: Not sure if this was intentional, but since we were talking about changes in the format yesterday - we currently do `Latest Version | Release Date`, as opposed to `Release Date | Latest Version`